client/web/src/index.tsx

```diff
@@ -2,6 +2,9 @@ import React from "react"
 import { createRoot } from "react-dom/client"
 import App from "src/components/app"
 
+declare var window: any;
+window.Tailscale = true
```

Comment:
Add a little doc that this is used to determine if the react client is built?

Reply:
will do.  Also, is there a better way to do this?  I couldn't come up with another way to detect if we have the client assets.

Reply:
done